Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -8,14 +8,16 @@
 
 import pyqtgraph as pg
 import Orange.data
+from Orange.data import Table, Domain, ContinuousVariable, StringVariable
```

Comment:
I don't think the above `import Orange.data` is necessary, since you are importing everything here. Also in this case replace all instances of `Orange.data.Table` with `Table`.